### PR TITLE
Added new content and initialized pages

### PR DESCRIPTION
**Pull request by hydraadra112**
- Created NavBar and Footer
- Added assets for navbar (stored in assets and code stored in components (BurgerButton and Cart))
- Initialized Pages (saved in `src/pages/`) by putting a navbar and footer
- Created dropdown for burger icon

### CODE REVIEW

File: package.json
Comment (computerscience-person) [inline package.json:1]: Remove nalang guro anay react-router from package.json.

Comment (hydraadra112) [reply package.json:1]: I'll remove it if maka push ako liwat haha

Comment (computerscience-person) [reply package.json:1]: I think we can do it okay with Vite

Comment (hydraadra112) [reply package.json:1]: okies ahh
ikaw man lead dev ahahaha

File: package.json [hunk 1]
```diff
@@ -15,7 +15,8 @@
   },
   "dependencies": {
     "react": "^18.3.1",
-    "react-dom": "^18.3.1"
+    "react-dom": "^18.3.1",
+    "react-router-dom": "^6.26.2"
```
Comment (computerscience-person) [inline package.json]: Remove this po

**Review by computerscience-person — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (computerscience-person) [reply package.json]: ang react-router-dom

Comment (hydraadra112) [reply package.json]: aight ill remove it directly sa github nlng haha

Comment (hydraadra112) [reply package.json]: done po

**Comment by computerscience-person** [comment]
LGTM! I'll fix nalang if may ara issues.

**Comment by hydraadra112** [comment]
thanks. love u

**Pull request merged**